x-pack/plugins/alerting_builtins/server/alert_types/index_threshold/lib/time_series_query.ts

```diff
@@ -134,8 +135,8 @@ export async function timeSeriesQuery(
     esResult = await callCluster('search', esQuery);
   } catch (err) {
     // console.log('time_series_query.ts error\n', JSON.stringify(err, null, 4));
-    logger.warn(`${logPrefix} error: ${JSON.stringify(err.message)}`);
-    throw new Error('error running search');
```

Comment:
Instead of throwing an error, just return an empty result.  Also improve the log message (doesn't need to be stringified).